lib/ifc-utils.ts

```
@@ -1060,10 +1089,26 @@ export function manageProperties(
 
       case "set":
       case "add":
+        // Determine the value to use for this specific element
+        let valueToSet = propertyValue;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)